app/assets/javascripts/utils/course_utils.js

```diff
@@ -58,6 +58,19 @@ const CourseUtils = class {
       };
     }
 
+    const wikiSourceUrlParts = /wikisource\.org\/wiki\/([^#]*)/.exec(articleTitle);
```

Comment:
'wikisource' is one term, so changing the caps will avoid ambiguity in this name: `wikisourceUrlParts`